src/letsRoute.js

```diff
@@ -55,17 +72,7 @@ Anumargak.prototype._on = function (method, url, fn) {
         }
     }
 
-    var namedExpressionMatches = getNamedExpressionMatches(url, namedExpressionRegexStr);
-    if (namedExpressionMatches && namedExpressionMatches.length > 0) {
-        for (var i = 0; i < namedExpressionMatches.length; i++) {
-            var matchedNamedKey = namedExpressionMatches[i][1];
-            if (this.namedExpression[matchedNamedKey]) {
-                url = url.replace(":" + matchedNamedKey + ":", this.namedExpression[matchedNamedKey]);
-            } else {
-                throw Error("Usage of named expression in url as " + matchedNamedKey + ". Define it using addNamedExpression method before using in URLs.");
-            }
-        }
-    }
+    url = replaceNamedExpression(url, this);
```

Comment:
this line would be changed to `url = this.replaceNamedExpression(url);`

Reply:
I think  better to use bind as this method should not be exposed I guess. Let me do that and you review it.